lib/plugins/track.js

```diff
@@ -189,7 +198,7 @@ exports.configure = function configure (config) {
 
```

Comment:
Shouldn't we check up here if the value we're trying to set is `undefined` and either reject or call `removeCookie` ?